pom.xml

```diff
@@ -203,9 +203,9 @@
         </license>
     </licenses>
     <scm>
-        <connection>scm:git:https://github.com/JWebUnit/jwebunit.git</connection>
-        <developerConnection>scm:git:https://github.com/JWebUnit/jwebunit.git</developerConnection>
-        <url>https://github.com/JWebUnit/jwebunit</url>
+        <connection>scm:git:https://github.com/atlassian/jwebunit.git</connection>
+        <developerConnection>scm:git:https://github.com/atlassian/jwebunit.git</developerConnection>
```

Comment:
We prefer ssh git URIs for scm connection (used by maven-release-plugin for recursive checkout) and developerConnection (used by maven-release-plugin for push) in our other projects, rather than https URIs. This should be okay when releasing locally if you're configured to use https authentication already.